recipe/meta.yaml

```diff
@@ -30,9 +30,10 @@ requirements:
     - expat
     - gsl
     - krb5  # [not win]
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
```

Comment:
@czender, like I said on slack, I think these need to be in `build`, not `host`, but I could be wrong.  @ocefpaf?

Reply:
Poking around on conda-forge, they seem to be in *both* `build` and `host` on every package I could find.  I don't understand why that would be necessary.

Reply:
@czender, if the testing is happy and doing what you want, don't change anything for now.  I'll try copying these to `build` as part of a release PR.

Reply:
I'm having a hard time determining if OpenMP will be active in the conda executables. I receive sporadic (and unreproducible) reports of things like NCO UDUnits and OpenMP functionality not working for people in the E3SM-Unified environment. So I'm going to try to eliminate these warnings in the raw pipeline output by putting the OpenMP in both host and build sections.

Reply:
Okay, let's try that.  I know that I've had tons of trouble with conda-forge MPI on our HPC systems but maybe system OpenMP plays nicer with conda packages than MPI does?